panel/tests/widgets/test_stats_plot_card.py

```diff
@@ -0,0 +1,68 @@
+import random
+
+import panel as pn
+from bokeh.models.sources import ColumnDataSource
+from panel.widgets.stats_plot_card import StatsPlotCard
+
+
+def test_constructor():
+    return StatsPlotCard(title="Test")
+
+
+def test_app():
```

Comment:
I'd like to organize all your manual tests into one module or in fact make them into the examples and then add some explicit example tests. Doesn't have to be in this PR though.

Reply:
Ok. The first step here would be you @philippjfr determining how and where you want those examples organized. A first reference example would be a good idea and maybe a comment in the developer docs about it.